### DISCUSSION TITLE
Sequelize Integration

### DISCUSSION THREAD

**Discussion by smuralimohan** [adonisjs/core#3423] [General]
Hi, 

I am currently using  ExpressJS  with Sequelize ORM for my web app. I want to migrate to AdonisJs, but want to continue using Sequelize ,as I am reusing a lot of Sequelize Models  in my client-side app as well.  

Is there a provision to bypass Lucid and instead use Sequelize with AdonisJS? 

Inputs appreciated.

Thank you,
Murali Mohan

**Comment by thetutlage** [comment]
Lucid does not come pre installed with AdonisJS. Just ignore it and use Sequelize like you will do it with any other Node app.

**Reply by smuralimohan** [reply]
Awesome! Thanks Harminder.

**Reply by skeeith** [reply]
I've read the Custom Adapter, but I'm a little confuse, I wanted to use Sequelize as well as the default ORM.
Do you guys have an example code as how to properly implement Sequelize into AdonisJS.

It would've better if AdonisJS itself would have an official Driver Support to Sequelize.

**Reply by RomainLanz** [reply]
It would be better if you drop Sequelize and use Lucid. 😅

**Reply by skeeith** [reply]
I'm actually coming from Laravel, my concern is, if Lucid is as powerful as Eloquent itself?

**Reply by skeeith** [reply]
@RomainLanz btw, are you guys looking for more maintainers?

**Comment by skeeith** [comment]
Can we petition to have a 1st Party Driver support for Sequelize?

so we can just change the `.env` variable like `mysql_driver=sequelize`

**Reply by thetutlage** [reply]
Sequelize is an ORM and you can use it as you will use it with any other Node.js project. Just install the package, import it and use it
